trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/VerificationResultTransformer.java

```diff
@@ -345,7 +345,7 @@ private IProgramExecution<IAction, Term> reduceRtInconsistencyProgramExecution(f
 			if (hasInvalidBranchEncoders(pe)) {
 				mLogger.info("Computing branch encoders");
 				final TraceCheck<IAction> tcl =
-						TraceCheck.createTraceCheck(mServices, toolkit, mgdScriptTc, truePred, falsePred, trace);
+						TraceCheck.createTraceCheck(mServices, toolkit, mgdScriptTc, truePred, falsePred, trace, null);
```

Comment:
The control configurations that are the subject of this PR should not be used for the purpose of computing executions. Since they have type `Object`, the only thing that should be done for them is to do equality-comparisons (e.g. to detect loops when configurations repeat).

Reply:
Hm, so how are we going to compute rcfg computations in the future?

Reply:
Can you explain how the computation of ICFG executions is connected to this? I don't see it.

Reply:
You are right, its actually not needed. I thought it was when I was looking at TraceCheck recently, but I was wrong